lib/resolvers/PackageResolver.ts

```diff
@@ -105,6 +117,24 @@ export class PackageResolver {
     });
   }
 
+  /**
+   * Retrieves the latest Angular Material version remotely.
+   */
+  private static _retrieveLatestVersion(): Promise<string> {
+    return new Promise((resolve, reject) => {
+      request('http://material.angularjs.org/docs.json', (error, response, body) => {
```

Comment:
It would be better to rely on NPM here, since the `docs.json` is not a file we can rely on.

We should use NPM to figure the latest version out
```
npm view angular-material version
```